test/integration/audited_hacks_async_test.rb

```diff
@@ -81,6 +82,19 @@ class AuditedHacksAsyncTest < ActionDispatch::IntegrationTest
     end
   end
 
+  # this will hopefully remind us to disable the :touch callback in audited 5.3
+  test "async audit causes no synchronous DB queries" do
+    # just a model without noise producing callbacks
+    line_item = FactoryBot.build(:line_item)
+
+    assert_number_of_queries(2, matching: /./) do
+      LineItem.with_auditing do
+        line_item.save!
+        line_item.update!(quantity: 5)
```

Comment:
why save and update both?

can we remove line_item.save! or its added for a purpose